src/mixins/filing-template-mixin.ts

```diff
@@ -1,20 +1,26 @@
 // Libraries
 import { Component, Vue } from 'vue-property-decorator'
 import { Action, Getter, State } from 'vuex-class'
+
 // Interfaces
 import {
   ActionBindingIF,
+  AlterationFilingIF,
+  BusinessSnapshotIF,
   CorrectionFilingIF,
   IncorporationFilingIF,
   OrgPersonIF,
   ShareClassIF,
   StateModelIF,
   NameTranslationIF,
-  NameTranslationDraftIF
+  NameTranslationDraftIF,
+  RoleIF
```

Comment:
I think you don't actually use RoleIF (or Roles interface below) in this module -- can be deleted from imports.